charts/qat/templates/qat.yaml

```diff
@@ -0,0 +1,23 @@
+{{- /*
+based on
+deployments/operator/samples/deviceplugin_v1_qatdeviceplugin.yaml
+*/}}
+
+apiVersion: deviceplugin.intel.com/v1
+kind: QatDevicePlugin
+metadata:
+  name: {{ .Values.qat.metadata.name }}
+  annotations:
+    {{ toYaml .Values.qat.metadata.annotations | indent 4 }}
+spec:
+  image: {{ .Values.qat.spec.image }}
+  #initImage: {{ .Values.qat.spec.initImage }}
```

Comment:
I get an error when enabling the initImage field

Reply:
initImage fails or helm fails? instead of commenting out lines, it's better to do the templating like this (syntax not tested):

```
{{- if Values.qat.spec.initImage }}
    initImage: ...
{{- end}}